manifests/w/WinSCP/WinSCP/5.19/WinSCP.WinSCP.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WinSCP.WinSCP
+PackageVersion: 5.19
+MinimumOSVersion: 10.0.0.0
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  Scope: machine
+  InstallerUrl: https://sourceforge.net/projects/winscp/files/WinSCP/5.19/WinSCP-5.19-Setup.exe/download

```

Comment:
This should do it :)
```suggestion
  InstallerUrl: https://nav.dl.sourceforge.net/project/winscp/WinSCP/5.19/WinSCP-5.19-Setup.exe

```

Reply:
Hmm, we've been using the first sourceforge link without issues before. Is that a permanent dynamic download center link?

Reply:
I think the Original link is best to use
![image](https://user-images.githubusercontent.com/15158490/122521183-cd774080-d014-11eb-9b3e-3578fac35b6f.png)